src/app/courses/courses-routing.module.ts

```diff
@@ -0,0 +1,35 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { CoursesComponent } from './courses.component';
+import { AddCourseComponent } from './add-course/add-course.component';
+import { CoursesListComponent } from './courses-list/courses-list.component';
+
+const routes: Routes = [
+  {
+    path: "courses",
```

Comment:
We already define `'courses'` route in `app-routing.module.ts`.
And it looks like here we don't need this route and empty `CoursesComponent`.

I suggest making it simple, we could define the part from children only.

Something like that:

```
const routes: Routes = [
     {
        path: "",
        component: CoursesListComponent,
      },
      {
        path: "new",
        component: AddCourseComponent,
      },
....
```